docs/_documentations/troubleshooting.md

```diff
@@ -299,9 +299,10 @@ Issue link: https://github.com/eclipse/codewind/issues/1251
 Info added in 0.10.0.
 -->
 ## Setting Codewind server logging levels
-To assist with problem determination, change the default log level, such as trace, in PFE with the `cwctl loglevels` command.
+To assist with problem determination, raise the default Codewind server log level to **Debug** or **Trace** with the `cwctl loglevels` command.
```

Comment:
we should make it clear that the `cwctl loglevels` does the exact same thing as the IDE instructions below